modules/api/pom.xml

```diff
@@ -81,7 +81,6 @@
         <dependency>
             <groupId>com.intuit.autumn</groupId>
             <artifactId>autumn-api</artifactId>
-            <version>1.0.20160714060618</version>
```

Comment:
Yes @iizrailevsky ! Need to clean this in other poms as well. But it will be better to do it as a separate change as it is unrelated.